### PR TITLE
New package: drawing-1.0.1

### PR DESCRIPTION
**Pull request by mrkcee**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture (X86_64-GLIBC)

### CODE REVIEW

File: srcpkgs/drawing/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'drawing'
+pkgname=drawing
+version=1.0.1
+revision=1
+wrksrc=drawing-$version
```
Comment (classabbyamp) [inline srcpkgs/drawing/template:5]: this is the default value, it does not need to be specified

File: srcpkgs/drawing/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=meson
+build_helper=gir
+hostmakedepends="python3 glib-devel appstream-glib pkg-config gettext itstool"
+makedepends="libglib-devel gtk+3-devel"
+depends="python3-gobject python3-cairo"
+short_desc="Simple image editor, similar to Microsoft Paint, for GNOME desktop."
+maintainer="mrkcee <markz_a@yahoo.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/maoschanz/drawing"
+distfiles="$homepage/archive/refs/tags/${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/drawing/template:15]: expand `$homepage` here

File: srcpkgs/drawing/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+revision=1
+wrksrc=drawing-$version
+build_style=meson
+build_helper=gir
+hostmakedepends="python3 glib-devel appstream-glib pkg-config gettext itstool"
+makedepends="libglib-devel gtk+3-devel"
+depends="python3-gobject python3-cairo"
+short_desc="Simple image editor, similar to Microsoft Paint, for GNOME desktop."
+maintainer="mrkcee <markz_a@yahoo.com>"
+license="GPL-3.0-only"
```
Comment (classabbyamp) [inline srcpkgs/drawing/template:13]: [from what I can see](https://github.com/maoschanz/drawing/search?q=any+later), this should be `GPL-3.0-or-later`

**Comment by classabbyamp** [comment]
also, please squash this into one commit

**Comment by mrkcee** [comment]
@classabbyamp PR comments are resolved

**Comment by mrkcee** [comment]
Removed period in description based on linter error

File: srcpkgs/drawing/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'drawing'
+pkgname=drawing
+version=1.0.1
+revision=1
+build_style=meson
+build_helper=gir
+hostmakedepends="python3 glib-devel appstream-glib pkg-config gettext itstool"
+makedepends="libglib-devel gtk+3-devel"
+depends="python3-gobject python3-cairo"
```
Comment (paper42) [inline srcpkgs/drawing/template:9]: at least gtk is missing from depends

Comment (mrkcee) [reply srcpkgs/drawing/template]: Added gtk+3

**Pull request closed without merging**